Formula/slirp4netns.rb

```diff
@@ -0,0 +1,31 @@
+class Slirp4netns < Formula
+  desc "User-mode networking for unprivileged network namespaces"
+  homepage "https://github.com/rootless-containers/slirp4netns"
+  url "https://github.com/rootless-containers/slirp4netns/archive/refs/tags/v1.1.11.tar.gz"
+  sha256 "87a8909746781d995b1b49eb36540e6ee745599f983c18f9b4e927ec92d86eb6"
+  license "GPL-2.0-only"
+
+  depends_on "jq" => :test
+
+  depends_on "glib"
+  depends_on "libcap"
+  depends_on "libseccomp"
+  depends_on "libslirp"
+  depends_on :linux
+
+  resource "source-code" do
+    url "https://github.com/rootless-containers/slirp4netns/archive/refs/tags/v1.1.11.tar.gz"
+    sha256 "87a8909746781d995b1b49eb36540e6ee745599f983c18f9b4e927ec92d86eb6"
+  end
```

Comment:
We shouldn't add the source code as a resource to use it in the `test` block – it is intentionally inaccessible. Instead, it would be better to extract a test case from their tests for the `test` block.

Reply:
Thanks for the suggestion. I have updated it accordingly.